docs/usage/capabilities_list.md

```diff
@@ -0,0 +1,75 @@
+CapabilitiesList
+========================================
+
+Overview
+========================================
+
+**CapabilitiesList** is a Java application to list the capabilities
+of the "Mozilla-JSS" provider and other installed providers.
+It does it in two fashions brief and verbose. 
+
+It lets JSS contributors see what features the JDK implements (at different versions)
+and see what else we'd need to add in, to reach compatibility. Since we're 
+attempting to be a default crypto provider, it would be nice to ensure we're 
+mostly close to what the JDK provides. Hopefully other packages written to the
+JDK provider can have similar algorithms available under JSS and behave well. 
+Large gaps in coverage are likely an issue (such as #341 and #242).
```

Comment:
What do you mean by `linking these issues so it's clickable`

Reply:
You can create a link in Markdown like this:
```
such as [#341](https://github.com/dogtagpki/jss/issues/341) ...
```
It will appear like this:
> such as [#341](https://github.com/dogtagpki/jss/issues/341) ...

so you can click the link to open the issue.

Reply:
I do understand now: links to the two issues mentioned in line 16

Reply:
Thanks for the update, but it looks like the second link is not rendered correctly.

Reply:
@emaldona Here's the first remaining issue.

Reply:
[#341](https://github.com/dogtagpki/jss/issues/341) is okay and the
[#342] (https://github.com/dogtagpki/jss/issues/342) isn't due the extra space between the closing bracket and the opening parenthesis so changing it
[#342](https://github.com/dogtagpki/jss/issues/342) should fix it

Reply:
Not what I intended to type but I bet you understand